README.md

```diff
@@ -1,5 +1,54 @@
-# 2dTo3dVector
-## Additions
-A CLI to convert 2D vector data to 3D vector data. 
-Testing a commit with VSCode
-Now, submitting a Pull Request instead of a Commit
\ No newline at end of file
+# Mountaineer
+A CLI to convert 2D vector data to 3D vector data.  
+
+## Usage
+CLI Usage: One liner
+
+mountaineer [input file] [lat] [lon] [output file]
+
+[input file]: Input file. File type can be .KML, .SHP, .GeoJSON (preferred). These are geospatial formats that contain Lat/long coordinates. Input file length should not exceed 128 characters. 
+
+[lat]: Latitude field. Required to create WKT
+[lon]: Longitude field. Required to create WKT
+
+[output]: User-input Output file name. File will be placed in the same folder as the input file. File length should not exceed 128 characters. File output will DEFAULT to .GeoJSON, but other outputs can be selected by changing the file extension. 
```

Comment:
A better description would be "Output file path". If you want to support outputting different file types, then add an option for `--format=<shp|geojson|...>`.

Reply:
Will add this format type in a future revision.